test/helpers.ts

```diff
@@ -279,3 +280,31 @@ export function prettyPrintPuppeteerAssertionError(message) {
 		assert.equal(match[1], match[2]);
 	}
 }
+
+export async function getNewPage(browser: Browser): Promise<Page> {
+	const pages = await browser.pages();
+	if (pages.length) return pages[0];
+	const page = await browser.newPage();
+
+	page.on('console', (type) => {
+		console[type._type](type._text);
+	});
+
+	page.on('error', error => {
+		console.log('>>> an error happened');
+		console.error(error);
+	});
+
+	return page;
+}
+
+export async function retryAsync<T>(fn: () => Promise<T>): Promise<T> {
+	let attempts = 0;
+	while (attempts <= 3) {
```

Comment:
Can we do a `while (true)` here? So we can avoid the hardcoded `3` twice. And maybe `retryAsync` could accept another parameter for `attempts` too, if we ever need to change the default `3`.

Reply:
I added `maxAttempts` argument.
https://github.com/sveltejs/svelte/pull/7076/commits/6c397e8ccf9d11ea9b1422f279603f6493fa468c

> Can we do a while (true) here?

No. Lint error comes.

![image](https://user-images.githubusercontent.com/19153718/147842920-b0afa2b5-622f-4d25-b740-2a023ad108a9.png)